### PR TITLE
Make altars craftable

### PR DESCRIPTION
**Pull request by Vannn4**
## About the PR 
This PR makes all of the recipes craftable. The recipes are same for all of the altars since there is no gameplay difference between them at the moment.

**Changelog**

:cl:
- add: Added altars to the Construction menu
- fix: Fixed burden altar using the incorrect sprite

### CODE REVIEW

**Comment by Vordenburg** [comment]
Old test failures are showing for this even after re-running them. Merging master might help.

**Comment by Vannn4** [comment]
Im not sure how those errors are related to what i did

**Comment by Vannn4** [comment]
> Old test failures are showing for this even after re-running them. Merging master might help.

You mean i should update the branch?

**Comment by Vordenburg** [comment]
Yes. Switch into master on your local repo, pull the latest changes from this repo, then `git merge master` on your branch and push.

**Comment by Vannn4** [comment]
Might need a test rerun

**Comment by Vordenburg** [comment]
Queueing up now.

**Comment by Vannn4** [comment]
Good to go

**Comment by Vordenburg** [comment]
All tests passing. 😎

**Comment by Elijahrane** [comment]
This should be good for now. I think, perhaps, in the future it should be 1 graph and then changing sprite from the UI. That would also let them change their starting altar.

**Review by Elijahrane — APPROVED** [review]

File: Resources/Prototypes/Entities/Structures/Furniture/altar.yml [hunk 1]
```diff
@@ -254,16 +284,19 @@
...
     sprite: Structures/Furniture/Altars/Gods/convertaltar.rsi
     layers:
-    - state: convertaltar
+    - state: burden
     - state: lightning
       shader: unshaded
   - type: Icon
     sprite: Structures/Furniture/Altars/Gods/convertaltar.rsi
-    state: convertaltar
+    state: burden
```
Comment (Vordenburg) [inline Resources/Prototypes/Entities/Structures/Furniture/altar.yml:292]: This seems like an oversight that can be sent upstream.

**Review by Vordenburg — APPROVED** [review]
Looks pretty good. We might have to re-visit this some day if construction graphs become less verbose for these kind of cases of minor variations.

**Comment by Vordenburg** [comment]
> This should be good for now. I think, perhaps, in the future it should be 1 graph and then changing sprite from the UI. That would also let them change their starting altar.

Do you mean something like a system where you create a basic altar then can convert it along with any other types?

**Pull request merged**